scripts/build_tags_taxonomy.pl

```diff
@@ -25,14 +25,51 @@
 use Modern::Perl '2012';
 use utf8;
 
+use ProductOpener::Config qw/:all/;
 use ProductOpener::Tags qw/:all/;
 
 my $tagtype = $ARGV[0];
 my $publish = $ARGV[1];
 
 print "building taxonomy for $tagtype - publish: $publish\n";
 
-build_tags_taxonomy($tagtype, $publish);
+binmode STDERR, ":encoding(UTF-8)";
+binmode STDIN, ":encoding(UTF-8)";
+binmode STDOUT, ":encoding(UTF-8)";
+
+
+my $file = $tagtype . ".txt";
+
+# For the Open Food Facts ingredients taxonomy, concatenate additives, minerals, vitamins, nucleotides and other nutritional substances taxonomies
+
+if (($tagtype eq "ingredients") and ($server_domain =~ /openfoodfacts/)) {
+
+	$file = "ingredients.all.txt";
+		
+	open (my $OUT, ">:encoding(UTF-8)", "$data_root/taxonomies/$file") or die("Cannot write $data_root/taxonomies/$file : $!\n");
```

Comment:
Does the taxonomy parser work with synonyms and stopwords for a language potentially being defined several times?

Reply:
It should work.